src/workerd/util/sqlite.c++

```diff
@@ -584,6 +510,45 @@ bool SqliteDatabase::isAuthorized(int actionCode,
         return allowSet.contains(KJ_ASSERT_NONNULL(param2));
       }
 
+    default:
+      return false;
```

Comment:
I preferred the logic before, where we had `case` statements for all of the known cases, explicitly rejecting the ones we know we don't want to allow, and then had a `default` which would warn so we could find out if we were missing something. Now this is just `return false`, so we don't get a warning. Also `SQLITE_RECURSIVE` is now blocked here where it was allowed before -- is that OK?

More abstractly, it seems tricky to think about which actions apply to "main" vs. "null", and it's not clear that having separate logic for them really helps anything.

Can we perhaps walk back some of this, and keep the combined code for the "main" and "null" cases? It seems like we only really want a new, separate path for the "temp" case.